### PR TITLE
FEAT - Add 6963 support to injected wallets module

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
This PR adds 6963 support to injected wallets module

### Checklist
- [x] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [x] Check the box that allows repo maintainers to update this PR
- [x] Test locally to make sure this feature/fix works
- [x] Run `yarn check-all` to confirm there are not any associated errors
- [x] Confirm this PR passes Circle CI checks
- [x] Add or update relevant information in the documentation

### Docs Checklist
- [x] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [x] Add/update the appropriate package README (if applicable)
- [x] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [x] Add/update the related package in the `docs/package.json` file (if applicable)

### CODE REVIEW

**Review by leightkt — APPROVED** [review]

File: packages/demo/src/App.svelte [hunk 1]
```diff
@@ -602,6 +602,10 @@
   const updateTheme = () => {
     onboard.state.actions.updateTheme(selectedTheme)
   }
+
+  function isSVG(str) {
```
Comment (leightkt) [inline packages/demo/src/App.svelte:606]: is this to prevent SVG Javascript Execution?

Comment (Adamj1232) [reply packages/demo/src/App.svelte:606]: Its to handle both svg and and img icons. Svgs are returned with 6963 which is already handled in the W3O

**Comment by Adamj1232** [comment]
@SocketSecurity ignore npm/bigint-buffer@1.1.5
@SocketSecurity ignore npm/keccak@1.4.0
@SocketSecurity ignore npm/keccak@3.0.3
@SocketSecurity ignore npm/bufferutil@4.0.6
@SocketSecurity ignore npm/utf-8-validate@5.0.8
@SocketSecurity ignore npm/web3@1.10.0

File: packages/demo/src/App.svelte (hunk 1 above)
Comment (Adamj1232) [reply packages/demo/src/App.svelte:606]: I added a check on the listening of  wallet announcements - good call!

Comment (leightkt) [reply packages/demo/src/App.svelte:606]: awesome- thanks!!

**Review by leightkt — APPROVED** [review]

**Review by aaronbarnardsound — APPROVED** [review]
Looks good, nice work!

File: packages/injected/src/wallets.ts [hunk 2]
```diff
@@ -890,7 +890,7 @@ const wallets = [
   zeal,
   exodus,
   frontier,
-  metamask,
+  // metamask,
```
Comment (Adamj1232) [inline packages/injected/src/wallets.ts:893]:
```suggestion
  metamask,
```

**Pull request merged**